chromium_src/components/history/core/browser/history_backend.h

```diff
@@ -6,8 +6,9 @@
 #ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_HISTORY_CORE_BROWSER_HISTORY_BACKEND_H_
 #define BRAVE_CHROMIUM_SRC_COMPONENTS_HISTORY_CORE_BROWSER_HISTORY_BACKEND_H_
 
-#define GetHistoryCount  \
-  GetKnownToSyncCount(); \
+#define GetHistoryCount                       \
+  GetKnownToSyncCount();                      \
+  static const int kExpireDaysThreshold = 60; \
```

Comment:
will shadowing of the existing var work if you make this one constexpr? This way you wouldn't need `const int HistoryBackend::kExpireDaysThreshold;` in .cc

Reply:
thanks, this options works good, removed line from .cc

https://github.com/brave/brave-core/pull/21880/commits/2d3bb537b94a1a1f2418ef574b7a8fd6ebe27f39#diff-63bcc9217276ef7d621d55a397500a8fa8e59e1306f429e740bba0f3bf7dc784R11